docs/operations.rst

```diff
@@ -24,11 +24,6 @@ However some operations are not supported, like operations that
 implicitly cause dense structures, or numpy functions that are not
 yet implemented for sparse arrays.
 
-.. code-block:: python
-
-   np.svd(x)  # sparse svd not implemented
-
-
```

Comment:
I wouldn't remove this.

Reply:
For sure. I think if we include that it should be `np.linalg.svd` and we should make it clear that sparse svd is supported by Scipy.

Reply:
Thinking about it a bit more, because we're trying to highlight "numpy functions that are **not yet** implemented for sparse arrays", perhaps we could use `np.linalg.cholesky`?